packages/p2p/src/components/toggle-container/toggle-container.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type TToggleContainer = {
+    children: React.ReactNode;
```

Comment:
nit: should this be `TToggleContainerProps`?

Reply:
Hmm but based on https://github.com/farrah-deriv/deriv-app/pull/6#discussion_r1205969189, we dont need to put `Props` at the end 🙂 . CMIIW 🙏

Reply:
Generally the basic convention is `TComponentNameProps`, plus the type is not exported anyways so the type is not used anywhere else so the naming generally can be anything (can even name it `Props`) so that's why its `nit`, but generally for consistency we name it `TComponentNameProps`. Here's a really nice reference:
https://www.totaltypescript.com/where-to-put-your-types-in-application-code

Reply:
I deleted the type instead, since its only has `children` as props.